GOVERNANCE.md

```diff
@@ -1,46 +1,30 @@
-> We (Synthstrom) are trying to stay a little hands off with this tbh and let the community work it out. We've got people helping with PR's and general support, but I felt it wouldn't be that fair for us to also have people setting up strategy etc, at that point its more or less us and no point to this all, haha. But I guess it might pay for someone to take a bit more of a 'strategy' role and create some documents which outline all the avail UI - slotting in possible features into that and making sure nothing is given up too hastily.
-> Create communication methods for the community to feedback directly to the dev's. There is lots that can be done in that regards, just needs some peeps with time and desire to manage that sort of planning.
-> Also, the creator has to be wiling to give their code back to the community. Lots of things need to come together and it might take a little while for things to get clearer.
+# Development strategy
 
-Via @jamiefaye in Discord:
+In order to give everyone in the project clear expecatations and transparency on what is going on all development is organized in the form of Pull requests. This ensures visibility on the ongoing activities and prevents intransparent internal changes from disrupting community work on Pull request. 
 
-> There are two levels of [governance] here. What goes into the Community branch and what goes into the Official branch. The Official level is up to Synthstrom. My goal as a contributor will be to make stuff that eventually goes into Official, with Community as a waypoint.
+There is no official roamdap or project activities, instead all contirbutors are welcome to participate in the discussion and planning process as outlined in the [contribution guidelines](CONTRIBUTING.md). Members of the community can build informal working groups to achieve greater goals and work on them in the form of collaborative Pull requests.
 
-Via @litui in [Discord](https://discord.com/channels/608916579421257728/1107026299945496577/1117474217475199016):
 
-> I think at some point community will need to level-set on vision for the Deluge. Will it aim for more comprehensive complexity like a DAW or will it aim to shorten the path to production?  Are these the same or different goals? etc etc.
+# Release strategy
 
-> More features in a build doesn't necessarily make music-making easier. Sometimes simplicity is best.
+There are releases coming from the ```develop``` branch as beta as well as the ```community``` branch as official releases. 
 
-### Stability First and Foremost?
+For ```develop``` or beta releases:
+* All incoming changes come from merging Pull requests from the community.
+* For every merged Pull request a new beta firmware will be generated and published on GitHub Releases tagged as ```beta``` to allow users to test them.
```

Comment:
As discussed in Discord, I tend to think *every* merge is a bit overkill, especially considering the number of merges that simply don't even involve core code changes beyond formatting (workflow, DBT, e2 config, etc.). Jamie has suggested, and I concur, that nightly builds would be a good idea, rolling up all the day's commits.

Reply:
I will make a commit soon with update to nightly 👍 I wrote it with the expectation that merges occur less than once per day but at least currently that is not the case.